config-model/src/main/java/com/yahoo/searchdefinition/RankProfile.java

```diff
@@ -501,10 +501,10 @@ public int getNumSearchPartitions() {
                 : numSearchPartitions;
     }
 
-    public double getTermwiseLimit() {
+    public Optional<Double> getTermwiseLimit() {
```

Comment:
Consider using `OptionalDouble` instead of `Optional<Double>`.

Reply:
Fixed